src/openrct2/ride/Ride.cpp

```diff
@@ -6175,17 +6170,19 @@ CoordsXYZD ride_get_entrance_or_exit_position_from_screen_position(const ScreenC
     {
         auto mapX = stationStart.x;
         auto mapY = stationStart.y;
-        entranceMinX = mapX;
-        entranceMinY = mapY;
+        auto entranceMinX = mapX;
+        auto entranceMinY = mapY;
+        auto entranceMaxX = mapX;
+        auto entranceMaxY = mapY;
```

Comment:
This didn't exist before, I'm assuming you're playing safe or fixing a bug?

Reply:
It was previously a legacy c function with all the variable declarations at the start of the function.

Reply:
I meant that `entranceMax*` variables were not initialized to `mapX` and `mapY`

Reply:
oh right yeah they were initialised to mapX and mapY later in the function but need for them to exist from earlier so thought might as well do it here like this.

Reply:
Ok